### ISSUE TITLE
can i connect  with TidTcpserver ?

### ISSUE THREAD

**Issue by DRAMA22** [andremussche/DelphiWebsockets#6]
can i create client with this to connect to Tidtcpserver ?

**Comment by andremussche** [comment]
No, not directly (unless you process the http headers yourself in the
tcpserver :) )
you can use this library to connect to a http server which supports
websockets, so you initially connect with http and then upgrade to to raw
tcp bi-direct mode

2016-05-25 8:17 GMT+02:00 DRAMA22 notifications@github.com:

**Comment by DRAMA22** [comment]
"unless i process the http headers" ? dont get what you mean

**Comment by andremussche** [comment]
never mind
but in fact http is some text headers over tcp :)
so you _could_ write some http header processing yourself (but don't do
that)

what do you actually want to achieve?

2016-05-26 4:36 GMT+02:00 DRAMA22 notifications@github.com:

**Comment by DRAMA22** [comment]
i have build client and server application that client can send some commands to the server , i wanted to build client that works on browser and send commands to server from it i thought i can do this with html5 and web socket when i made couple of search your delphiwebsocket comes up on my search result so i thought i could do it using Your web-socket indy wrapper but seems other thing have to be done hope you publish away to achieve connecting websocket client to Tidtcpserver and exchange data if its possible

**Comment by andremussche** [comment]
you cannot simply connect websocket with tidtcpserver because websockets
start as a normal http request, but in theory you can do _some_ http
processing in your tidtcpserver but I think it is better to use a normal
http server (with websocket support) and use that besides your current
tidtcpserver.

But personally I won't use low level direct tcp anymore, but socket.io and
websockets instead (all low level tcp stuff is done already for you)
because you have full duplex (websockets) and command processing (socket.io)
out of the box

2016-05-26 9:42 GMT+02:00 DRAMA22 notifications@github.com:
